Assets/scripts/ViewFastBullet.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ViewFastBullet : ViewBullet
+{
+    public void Start()
+    {
+        Destroy(gameObject, 3);
```

Comment:
lets change hard coded 3 to be in the model class

Reply:
Done